docs/.vitepress/theme/style.css

```diff
@@ -115,3 +120,41 @@
   transition: color 0.25s;
 }
 
+.vp-doc th,
```

Comment:
Thanks! But this feels a bit too much. I rather wait for VP to fix it.

Reply:
I'll try to simplify it a bit, I just copy styles from pwa docs, we are not using a lot of cases (code blocks inside details)

Reply:
Can we keep the complexity in VP? I think we should try to avoid duplications across repos

Reply:
Sure

Reply:
Anthony, simplified to minimal changes